acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -250,7 +250,8 @@ public void start(final String processName) {
                       });
 
               if (!downstreamTrustOptions.isCaAuthRequired()) {
-                params.add("--downstream-http-tls-ca-auth-disabled");
+                params.add("--downstream-http-tls-ca-auth-enabled");
+                params.add("false");
```

Comment:
this should remain to "true".

Reply:
but - I only want to change the default if we're turning it off, right?

Reply:
you are right ... the tests were written considering it off.

Reply:
you can change it so that it reads (removing if condition altogether):

                params.add("--downstream-http-tls-ca-auth-enabled");
                params.add(downstreamTrustOptions.isCaAuthRequired());